src/metabase/models/setting.clj

```diff
@@ -274,6 +271,10 @@
    ;; where this setting should be visible (default: :admin)
    [:visibility Visibility]
 
+   ;; should this setting be encrypted `:never` or `:maybe` (when `MB_ENCRYPTION_SECRET_KEY` is set).
+   ;; Defaults to `:maybe`
+   [:encryption [:enum :never :maybe]]
```

Comment:
Not a blocker but perhaps we should put defaults into enums and use them from the caller, if it's not too awkward